Python/makeopcodetargets.py

```diff
@@ -1,4 +1,3 @@
-#! /usr/bin/env python
```

Comment:
I don't think makeopcodetargets needs a locally built Python. Does it?

Reply:
you are right. It will be called in makefile: https://github.com/python/cpython/blob/master/Makefile.pre.in#L946.
So I think the shebang haven't worked in here.